tests/functional/reporting-non-auth.js

```diff
@@ -73,88 +73,36 @@ define(
       },
 
       "URL validation": function() {
-        return FunctionalHelpers.openPage(
-          this,
-          url("/issues/new"),
-          ".wc-ReportForm-actions-button"
-        )
-          .findByCssSelector("#url")
-          .click()
-          .end()
-          .execute(function() {
-            var elm = document.querySelector("#url");
-            WindowHelpers.sendEvent(elm, "input");
-          })
-          .sleep(500)
-          .findByCssSelector(".wc-Form-helpMessage")
-          .getVisibleText()
-          .then(function(text) {
-            assert.include(
-              text,
-              "A valid URL is required",
-              "URL validation message is shown"
-            );
-          })
-          .end()
-          .findByCssSelector("#url")
-          .type("sup")
-          .end()
-          .waitForDeletedByCssSelector(".wc-Form-helpMessage")
-          .end();
-      },
-
-      "(optional) browser + os validation": function() {
         return (
           FunctionalHelpers.openPage(
             this,
             url("/issues/new"),
             ".wc-ReportForm-actions-button"
           )
-            // make sure we can see the valid checkbox (i.e. it's background image is non-empty)
-            .execute(function() {
-              return window
-                .getComputedStyle(
-                  document.querySelector(
-                    "div.wc-Form-group:nth-child(2) > span:nth-child(2)"
-                  ),
-                  ":after"
-                )
-                .getPropertyValue("background-image");
-            })
-            .then(function(bgImage) {
-              assert.include(
-                bgImage,
-                "valid.svg",
-                "The valid checkbox psuedo is visible"
-              );
-            })
+            .findByCssSelector("#url")
+            .click()
             .end()
             .execute(function() {
-              var elm = document.querySelector("#os");
-              elm.value = "";
+              var elm = document.querySelector("#url");
               WindowHelpers.sendEvent(elm, "input");
             })
-            .end()
             .sleep(500)
-            // make sure we can't see the valid checkbox (i.e. it's background image is empty)
-            .execute(function() {
-              return window
-                .getComputedStyle(
-                  document.querySelector(
-                    "div.wc-Form-group:nth-child(3) > span:nth-child(2)"
-                  ),
-                  ":after"
-                )
-                .getPropertyValue("background-image");
-            })
-            .then(function(bgImage) {
-              assert.notInclude(
-                bgImage,
-                "valid.svg",
-                "The valid checkbox psuedo is not visible"
+            // using xpath as workaround for timeout issue when error clears
```

Comment:
nit: this comment should probably be removed, since it isn't using findbyxpath (or whatever the method is called)